deploy/test-environments/main.tf

```diff
@@ -4,11 +4,12 @@ provider "aws" {
 
 locals {
   common_tags = {
-    division = "${var.division}"
-    org      = "${var.org}"
-    team     = "${var.team}"
-    project  = "${var.project}"
-    owner    = "${var.owner}"
+    division   = "${var.division}"
+    org        = "${var.org}"
+    team       = "${var.team}"
+    project    = "${var.project}"
+    owner      = "${var.owner}"
+    deployment = "${var.deployment_name}"
```

Comment:
do we need to populate `deployment_name` in the Github action?

Reply:
No, this has already been done [here](https://github.com/elastic/cloudbeat/blob/main/.github/workflows/test-environment.yml#L230).

Reply:
Nice